test/parallel/test-util-inspect.js

```diff
@@ -495,6 +495,13 @@ assert.strictEqual(util.inspect(-5e-324), '-5e-324');
   assert.strictEqual(util.inspect(a), "[ 'foo', 'bar', 'baz' ]");
   delete a[1];
   assert.strictEqual(util.inspect(a), "[ 'foo', <1 empty item>, 'baz' ]");
+  a.push('qux');
+  a.splice(3, 0, undefined);
+  delete a[3];
+  assert.strictEqual(util.inspect(a, {
+    maxArrayLength: 2
+  }), "[ 'foo', <1 empty item>, ... 3 more items ]");
```

Comment:
Ideally this could be moved down to be the very last test case. That way there are already enough entries in the array and it would work without pushing / splicing / deleting entries. Calling splice should definitely not be required.

(So below here https://github.com/nodejs/node/blob/5ce3d10434e0d81934f7013a2a13d1502be6aca6/test/parallel/test-util-inspect.js#L520)

Reply:
Ah, I understand.
I will redo.
`.splice` is heavy :)

Reply:
Moved too the end of sparse array tests.
Made coverage check for line 1183 again, it is there/
![image](https://user-images.githubusercontent.com/1651439/58649488-e7059c80-8314-11e9-8003-24d867270473.png)